pkg/autodetect/main.go

```diff
@@ -131,6 +131,30 @@ func (b *Background) detectCronjobsVersion(ctx context.Context) {
 	)
 }
 
+func (b *Background) detectAutoscalingVersion(ctx context.Context) {
+	apiGroupVersions := []string{"autoscaling/v2", "autoscaling/v2beta2"}
+	for _, apiGroupVersion := range apiGroupVersions {
+		groupAPIList, err := b.dcl.ServerResourcesForGroupVersion(apiGroupVersion)
+		if err != nil {
+			log.Log.V(-1).Info(
+				fmt.Sprintf("Error getting %s api list: %s", apiGroupVersion, err),
+			)
+			continue
+		}
+		for _, api := range groupAPIList.APIResources {
+			if api.Name == "horizontalpodautoscalers" {
+				viper.Set(v1.FlagAutoscalingVersion, apiGroupVersion)
+				log.Log.V(-1).Info(fmt.Sprintf("Found the horizontalpodautoscalers api in %s", apiGroupVersion))
```

Comment:
lets start error messages with a lower case letter :)

Reply:
@iblancasa there are a few more error messages starting with an upper case letter ^^

Reply:
Let's see if now is correct